Symfony/src/Entity/Posts.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\PostsRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use App\Entity\Users;
+
+#[ORM\Entity(repositoryClass: PostsRepository::class)]
+class Posts
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $title = null;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $text = null;
+
+    #[ORM\Column(type: Types::JSON, nullable: true)]
+    private ?array $photo;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $author = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
+    private ?\DateTimeInterface $created_date = null;
```

Comment:
Attention aux conventions de nommage PHP, les attributs ce n'est pas du snake_case. Ici, pour nommer correctement ta colonne de BDD et ton attribut, tu peux faire ça : 

```suggestion
    #[ORM\Column(name: 'created_date', type: Types::DATETIME_MUTABLE)]
    private ?\DateTimeInterface $createdDate = null;
```

Reply:
les conventions dépendent du projet mais très rarement du langage..